plugins/module_utils/podman/podman_container_lib.py

```diff
@@ -976,7 +976,7 @@ def diffparam_log_level(self):
         if '--log-level' in excom:
             before = excom[excom.index('--log-level') + 1].lower()
         else:
-            if self.module.params['log_level'] is not None:
+            if 'log_level' in self.module_params:
```

Comment:
we still need to check if the value is not none.

Reply:
see my comment in PR